gridstatus/ercot.py

```diff
@@ -2274,7 +2303,13 @@ def _get_documents(
         return matches
 
     def _get_hourly_report(
```

Comment:
The github diff makes it seem like I changed this method, but the changes are in `parse_doc` to account for `TimeEnding` instead of `HourEnding` in the system-wide load.

Reply:
add the `TimeEnding` is what i misread. thanks for clarifying!